src/modal/test/modal.spec.js

```diff
@@ -131,6 +131,16 @@ describe('$uibModal', function() {
           elem.focus();
         }
       };
+    }).component('fooBar', {
+      bindings: {
+        resolve: '<',
```

Comment:
do we need to mention in the docs that this and the modal instance are one-way bound?  realizing that it doesn't apply to object contents...